src/lib/test-outbound/outbound-initiator.js

```diff
@@ -48,7 +46,7 @@ const dbAdapter = require('../db/adapters/dbAdapter')
 
 var terminateTraceIds = {}
 
-const getTracing = (traceID) => {
+const getTracing = (traceID, dfspId) => {
   const tracing = {
     outboundID: traceID,
     sessionID: null
```

Comment:
I think we can remove the dependency on the dfspId with session ID. A new session will be created each time when the user opens the webpage.